### DISCUSSION TITLE
Make saving replays of failed scores be exported automatically to match stable

### DISCUSSION THREAD

**Discussion by gowon11** [ppy/osu#30797] [Ideas] [answered]
Upon failing on a map we can save the replays of that play. On stable it direcly exports the replay file but on lazer it just saves the replay on the local scores. It would be nice if it was also exported so that the extra step of selecting said score on the list then exporting manually would be skipped.

Also maybe have a dedicated folder for exported replays.

**Comment by Joehuu** [comment]
F2 saves and exports.

Probably need a hotkey hint (like in stable) and also a button for mobile/touchscreen users.

**Reply by gowon11** [reply]
It exports if F2 is pressed on the result screen but not on the failed screen which is the point of my post.

**Reply by Joehuu** [reply]
Hmm... it shows it as a notification (like in stable iirc).

![Kapture 2024-11-20 at 14 18 41](https://github.com/user-attachments/assets/02245c81-7311-4f4e-81f8-1aad54e398ed)

**Reply by gowon11** [reply]
This is how the game behaves when i try to do it. I had to press F2 4 times until it was exported. I don't know how you did this overlay of the press of F2 so i tried to indicate my key presses with the mouse cursor. 

https://github.com/user-attachments/assets/38bd15fc-02b5-484d-94ef-ef994b77de06

**Reply by Joehuu** [reply]
Can reproduce on an install with many beatmaps downloaded (100k+ diffs). Will investigate.

**Reply by Joehuu** [reply, marked as answer]
This issue is also related to https://github.com/ppy/osu/issues/28497. Will post my findings there, as I dunno the right fix.
